### DISCUSSION TITLE
Customize Chart's Tooltip's Labels

### DISCUSSION THREAD

**Discussion by Kuuchuu** [Budibase/budibase#4555] [Help]
Currently the tooltip for my chart shows the column name as the label for the Y value, the column name being "ItemNetSales". Is there any way to change what it shows so that I can have it just display "Net Sales" instead?
Thanks

**Comment by mjashanks** [comment]
There's no way to change this. I think that the tooltip should display the same text as the "Y-Axis Label". But there's no way around this at the moment.

**Reply by Kuuchuu** [reply]
Gotcha, would this potentially be a bug then? The Y-Axis Label is "Sales" while the tooltip is the data's column's name.
![image](https://user-images.githubusercontent.com/20036947/154735811-03433223-20f1-4ede-b862-dea95d245384.png)
![image](https://user-images.githubusercontent.com/20036947/154736006-22266c31-ab1e-4126-9306-9491d9fb5cab.png)
